lib/geordi/commands/branch.rb

```diff
@@ -6,7 +6,9 @@
 stored in `~/.config/geordi/global.yml`.
 LONGDESC
 
+option :from_maser, aliases: '-m', type: :boolean, desc: 'Checkout new branch from master instead of the current branch'
```

Comment:
Please keep the description shorter:
```suggestion
option :from_master, aliases: '-m', type: :boolean, desc: 'Branch from master instead of the current branch'
```